src/components/manage/Controlpanels/AddonsControlpanel.jsx

```diff
@@ -218,6 +218,20 @@ class AddonsControlpanel extends Component {
     this.setState({ isClient: true });
   }
 
+  /**
+   * Component will receive props
+   * @method componentWillReceiveProps
+   * @param {Object} nextProps Next properties
+   * @returns {undefined}
+   */
+  UNSAFE_componentWillReceiveProps(nextProps) {
+    if (this.props.addonsLoading && nextProps.addonsError) {
+      this.setState({
+        error: nextProps.addonsError,
```

Comment:
What is the reason for setting `error` in the component state instead of simply using props.addonsError in the render function?